js_sdk/install_glia_with_site_id.js

```diff
@@ -0,0 +1,17 @@
+var installGlia = function(siteId, callback) {
+  const gliaIntegrationScriptUrl = `https://api.salemove.com/salemove_integration.js?site_id=${siteId}`;
+  const scriptTagName = 'script';
+  var scriptElement = document.createElement(scriptTagName);
+
+  scriptElement.async = 1;
+  scriptElement.src = gliaIntegrationScriptUrl;
+  scriptElement.type = "text/javascript";
+  scriptElement.addEventListener("load", callback);
```

Comment:
here I'd do
```js
if (typeof(callback) === 'function') {
  scriptElement.addEventListener("load", callback);
}
```
to make sure it doesn't fail if client doesn't specify the function